monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/ResultsFolderController.java

```diff
@@ -49,16 +51,52 @@
     @RolesAllowed(SecurityRoles.VIEWER)
     @RequestMapping(method = RequestMethod.GET, produces = "application/json")
     @ResponseBody
-    public List<Map<String, String>> resultsFolderJson(@PathVariable("tenant") final String tenant) {
+    public List<Map<String, String>> resultsFolderJson(@PathVariable("tenant") final String tenant,
+            @RequestParam(value = "time", required = false) final String timestamp) {
```

Comment:
Name here is a bit weird. Wouldn't "not_before", or something to that effect, better match the purpose?